### PR TITLE
gui: Seperate rpcconsole from nodewindow

### PR DESCRIPTION
**Pull request by RandyMcMillan**
Since the window is named "Node Window" the .ui file should reflect this.

### CODE REVIEW

File: src/qt/forms/nodewindow.ui [hunk 1]
```diff
@@ -6,12 +6,12 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>740</width>
-    <height>430</height>
+    <width>777</width>
```
Comment (promag) [inline src/qt/forms/nodewindow.ui:9]: Unrelated change.

**Review by promag — COMMENTED** [review]
Concept -0. File rename was avoided when the title was changed, see https://github.com/bitcoin/bitcoin/pull/17096.

**Comment by Sjors** [comment]
This change is more suitable for a larger PR that changes something meaningful in node window, see [developer notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-general):

> Do not submit patches solely to modify the style of existing code.

**Comment by RandyMcMillan** [comment]
@Sjors - the idea was to move toward restructuring the rpcConsole implementation to a "data_model view controller" type implementation - as it is now the "Node Window" seems too monolithic - the information, console, network traffic and peers tabs should be separated to follow a better design paradigm. I believe this is an impediment to better GUI design.

**Comment by hebasto** [comment]
@RandyMcMillan 
> @Sjors - the idea was to move toward restructuring the rpcConsole implementation to a "data_model view controller" type implementation - as it is now the "Node Window" seems too monolithic - the information, console, network traffic and peers tabs should be separated to follow a better design paradigm. I believe this is an impediment to better GUI design.

Nice topic for the [Bitcoin Design Community](https://bitcoindesign.slack.com)!

**Comment by promag** [comment]
@RandyMcMillan is the file rename a blocker?

**Comment by Sjors** [comment]
This commit can be part of a PR that actually does the restructuring.

**Comment by RandyMcMillan** [comment]
Implementing the change...thanks for the feed back.
I wasn't sure whether to do this change incrementally.
More to follow.
